mon-pix/app/components/navbar-desktop-header.js

```diff
@@ -13,7 +13,7 @@ export default class NavbarDesktopHeader extends Component {
 
   get menu() {
     const menuItems = this._menuItems;
-    return this.isUserLogged ? menuItems.filterBy('permanent', true) : menuItems;
+    return (this.isUserLogged || this._isFromExternalPlatform) ? menuItems.filterBy('permanent', true) : menuItems;
```

Comment:
Simplifier.
Par exemple :  
```suggestion
    return (this.isUserLogged || this._isFromExternalPlatform) ? [] : menuItems;
```

Reply:
Done.